var/spack/repos/builtin/packages/qd/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```

Comment:
Did this get sucked in from another PR?

Reply:
Yes, the QD package was added by another PR.  I've merged the latest develop to resolve it.